extensions/unicode-symbols/scripts/generate-dataset.ts

```diff
@@ -59,20 +63,45 @@ function getCharactersByCodeRange(startCode: number, endCode: number) {
   return characters;
 }
 
+const mapCodeToName = (char: Character): Character => {
+  const map: Record<number, string> = { 63743: "APPLE LOGO" };
+  return {
+    ...char,
+    name: map[char.code] || char.name,
+  };
+};
+
+function mapCharacterToDatasetItem(char: Character) {
+  return { value: String.fromCodePoint(char.code), code: char.code, name: char.name };
+}
+
+function sanitizeCharacters(characters: Character[]) {
+  return characters
+    .filter(Boolean) // Include only valid characters
+    .map(mapCodeToName)
+    .map(mapCharacterToDatasetItem)
+    .filter((char) => char.name !== "<control>"); // Exclude invisible control characters
+}
+
 // Run the dataset generation.
 (function generateDataset() {
   const filteredBlocks = allBlocks.filter((block) => blockNamesToFilter.includes(block.blockName));
+  const additionalBlocks = allBlocks.filter((block) => additionalBlockNames.includes(block.blockName));
 
   const characters = filteredBlocks.flatMap((block) => {
-    return getCharactersByCodeRange(block.startCode, block.endCode)
-      .filter(Boolean) // Include only valid characters
-      .filter((char) => char.name !== "<control>") // Exclude invisible control characters
-      .map((char) => ({ value: String.fromCodePoint(char.code), code: char.code, name: char.name }));
+    return sanitizeCharacters(getCharactersByCodeRange(block.startCode, block.endCode));
   });
 
-  const dataset = { blocks: filteredBlocks, characters };
+  const additionalCharacters = sanitizeCharacters(
+    allCharacters.filter((char) => additionalCharacterValues.includes(String.fromCodePoint(char.code)))
+  );
+
+  const dataset = {
+    blocks: [...filteredBlocks, ...additionalBlocks],
+    characters: [...characters, ...additionalCharacters],
+  };
 
-  fs.writeFileSync(datasetOutputPath, JSON.stringify(dataset));
+  fs.writeFileSync(datasetOutputPath, JSON.stringify(dataset, null, 2));
```

Comment:
Just FYI, if I recall correctly the file was not being formatted here to keep the file size down since it's shipped with the extension. I'm OK with it, though 👍

Reply:
The one currently on `main` is formatted but with one line per entry, I'm not sure how it was generated since running this command without my change would put everything on a single line instead.

Reply:
Yeah, I think I might have formatted it with prettier by mistake.